src/OneDriveSdk/Requests/Generated/IChildrenCollectionRequest.cs

```diff
@@ -33,9 +33,9 @@ public partial interface IChildrenCollectionRequest : IBaseRequest
     {
         
         /// <summary>
-        /// Adds the specified Item to the collection via POST.
```

Comment:
Let's talk offline for generated file feedback. We'd need to make changes in the template writer.